Live Tail for Live Containers (#4069)
DataDog/documentation: content/en/graphing/infrastructure/livecontainers.md

While actively working with the containers page, metrics are collected at a 2-second resolution. This is important for highly volatile metrics such as CPU. In the background, for historical context, metrics are collected at 10s resolution.

## Include/Exclude containers

> *Please note that Live Containers is not metered.  There is no reason to exclude containers for billing purposes.*

Reviewer: ```suggestion
*Note that Live Containers is not metered. Including or excluding containers does not affect billing.*
```
